src/Engine/Graphics/Renderer/OpenGLRenderer.cpp

```diff
@@ -4999,6 +5005,17 @@ void OpenGLRenderer::ReloadShaders() {
     }
 }
 
+void OpenGLRenderer::beginOverlays() {
+    ImGui_ImplOpenGL3_NewFrame();
+    openGLContext->startOverlayFrame();
+    ImGui::NewFrame();
+}
+
+void OpenGLRenderer::endOverlays() {
+    ImGui::Render();
+    ImGui_ImplOpenGL3_RenderDrawData(ImGui::GetDrawData());
+}
```

Comment:
Compared to Nuklear we have a similar approach to other begin/end functions in the renderer.
We'll get rid of Renderer::drawOverlays when we remove nuklear.